src/ImageSharp/Colors/Color.cs

```diff
@@ -70,28 +70,6 @@ public Color(byte r, byte g, byte b, byte a = 255)
         /// <summary>
         /// Initializes a new instance of the <see cref="Color"/> struct.
         /// </summary>
-        /// <param name="hex">
```

Comment:
This is great! Being able to create any color from a hex is a powerful addition.